fastlane/spec/fixtures/plist/Info-expected.plist

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
added a new plist since it was easier to validate against

Reply:
How come?

Reply:
Did because `get_info_plist_value` action only returns the value, and I was lazy 😐 

I'll try and rewrite the test to use expected objects instead of an entire plist.